pkgs/applications/misc/lutris/default.nix

```
@@ -31,7 +31,7 @@ let
     graphite2 gtk2 gtk3 udev ncurses wayland libglvnd vulkan-loader
 
     # Lutris
-    gobjectIntrospection gdk_pixbuf pango openssl sqlite xterm libnotify procps
+    gobjectIntrospection gdk_pixbuf hicolor-icon-theme pango openssl sqlite xterm libnotify procps
```

```suggestion
    gobject-introspection gdk_pixbuf hicolor-icon-theme pango openssl sqlite xterm libnotify procps
```

That's an old alias.

---

applied changes. Not really in scope of the commit's intent, but i guess it's better than having obsolete references around.